reference-implementations/icrc25/signer/src/agent.js

```diff
@@ -23,60 +23,41 @@ export async function call(agent, canisterId, methodName, arg) {
   return await polling.pollForResponse(agent, Principal.from(canisterId), callResponse.requestId, polling.defaultStrategy())
```

Comment:
`polling.defaultStrategy()` here is redundant

Reply:
It is not, for some reason `pollForResponse` does _not_ set a default if none is provided.

Reply:
Ah, you're right, my bad. For some reason I haven't noticed it was the library function being executed here, not the custom one some lines below.